src/sync-utils/sync-db.ts

```diff
@@ -336,10 +382,20 @@ const RECORD_SYNC_LOGIC: {
 
 export function computeRecordActions(
   deviceRecord: RawPdbRecord | null,
-  desktopRecord: RawPdbRecord | null
+  desktopRecord: RawPdbRecord | null,
+  slowSync: Boolean = false
```

Comment:
Similar to `string`, we should use lowercase `boolean` instead of `Boolean`. But actually in this case we can just say `slowSync = false` and TypeScript will automatically type this argument as `boolean`.

Reply:
Ohh I see, done!